docs/install/index.md

```diff
@@ -3,36 +3,29 @@ html_meta:
   "description": "Install Plone 6"
   "property=og:description": "Install Plone 6"
   "property=og:title": "Install Plone 6"
-  "keywords": "Plone 6, install, installation, installing, buildout, pip, docker"
+  "keywords": "Plone 6, install, overview"
 ---
 
 (install-index-label)=
 
 # Install Plone 6
 
-This section focuses on installing Plone 6 for the developer, tester, or contributor.
 
-For trying out Plone 6, visit our official demonstration site located at:
+(install-index-getting-started-label)=
 
-```{toctree}
-:maxdepth: 2
-:hidden: true
+## Getting started
 
-containers/index
-```
+* What do you want to do?
+  * Try a demo now
+  * Run Plone in containers (why use containers here..)
+  * Develop Plone add-ons or contribute to Plone packages → Install Plone from source
 
-https://6.demo.plone.org/
 
-```{todo}
-For deployment of Plone 6, visit TBD.
-```
-
-
-(install-index-system-requirements-pre-requisites-label)=
+(install-index-system-requirements-label)=
 
-## System requirements and pre-requisites
+## System Requirements
```

Comment:
## System Requirements for an installation from source.

Reply:
Do we have two different sets of system requirements, one for source install and one for containers? If so, then this section should be moved to the correct page, and replaced with "It depends on your choice of installation method."

Reply:
See 51f3f20a64781919bd3e7ba5c90530c1497bd6fc